### DISCUSSION TITLE
Show shortcut keys in tooltips

### DISCUSSION THREAD

**Discussion by xahodo** [vkbo/novelWriter#1155] [Ideas]
For a lot of functionality a shortcut key is available. It would be helpful with memorization of those shortcut keys if they show in the tooltips.

**Comment by vkbo** [comment]
I agree this is useful, and I've already started doing this with the new features added for the next release. They are generally also available in the menus. There are still a bunch of them that don't have a button or menu entry for where to add such info. Did you have any specific ones in mind?

**Reply by xahodo** [reply]
Well, I noticed it with tooltips all around the interface, so I thought I'd suggest it. I wasn't thinking about any one of them in particular.

PS: Sorry for erring with the github chat.

**Reply by vkbo** [reply]
No worries about the thread. It just gets potentially messy as I've noticed in other threads. I'm trying to avoid this.

Anyway, as mentioned I've already started adding shortcut info to tooltips where the shortcut is not mentioned elsewhere (like in a menu). When the shortcut *is* mentioned in a menu somewhere, the tooltip usually doesn't have it. Some features are also pure shortcuts and don't have a button or menu entry where the info can be put. As far as I can see in the development version of the code, the info is not missing anywhere where it could be added. In the released version there may be a couple of places perhaps, so it would really be helpful if you could name them so I can check that they are fixed with the next release.

There is also a feature request for a tool to change shortcuts in #740. If I do add this, all shortcuts will be listed there. As it is now, they are listed in the [documentation](https://novelwriter.readthedocs.io/en/latest/usage_shortcuts.html).

**Reply by xahodo** [reply]
- Project settings
- writing statistics
- Project details
All of them below the binder (as something similar is called in scrivener). Other things don't seem to have tooltips, but then again kde neon might have broken tooltips. Will check in Windows soon.

**Reply by vkbo** [reply]
Those have their shortcuts listed in the menu under "Project" and "Tools".
